feed/templates/feed/post.html

```diff
@@ -38,7 +38,7 @@ <h5><a class="text-decoration-none text-dark" href="{% url 'post-detail' post.id
                 {% if post.is_job_offer %}
                     <p>
                         <b>Job Type:</b> {{ post.prefernces.job_type.text }} <br>
-                        <b>Experience Level:</b> {{ post.prefernces.years_of_experience.text }} <br>
+                        <b>Experience Level:</b> {{ post.prefernces.years_of_experience }} <br>
```

Comment:
Do you think a test like this 
Can test if it is empty content?
```
self.assertIn(b'<title>To-Do lists</title>', response.content) 
```
is not
<b>Experience Level: <>

Reply:
I see, it is not going to be done